doc/releases/release-notes-3.2.rst

```diff
@@ -291,6 +311,9 @@ Drivers and Sensors
 
 * I2C
 
+  * Improve gd32 driver stability
```

Comment:
```suggestion
  * Improve gd32 driver stability (missing: how?)
```

Reply:
Add detail, repeated START condition have been removed, since there doesn't have any official description for this condition.